astronomer/providers/amazon/aws/hooks/redshift_data.py

```diff
@@ -77,6 +77,9 @@ def get_conn_params(self) -> Dict[str, Union[str, int]]:
                 if "secret_access_key" in extra_config
                 else extra_config["aws_secret_access_key"]
             )
+        elif connection_object.login:
+            conn_params["aws_access_key_id"] = connection_object.login
+            conn_params["aws_secret_access_key"] = connection_object.password
```

Comment:
between this method andhttps://github.com/astronomer/astronomer-providers/blob/main/astronomer/providers/amazon/aws/hooks/base_aws.py#L33 there is lot of redundant code we should see if we can improve it

Reply:
Yes that's async hook which gets the connection info this `redshift_data` hook  class

Reply:
We can fix this redundant code

Reply:
created ticket https://app.zenhub.com/workspaces/airflow-providers-team-61a89ee7fbf048001dc8378d/issues/astronomer/astronomer-providers/747